### ISSUE TITLE
expandable tile: can't reach below the fold content navigating in reverse from button using VO/NVDA

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#5181]
## What package(s) are you using?

- [ ] `carbon-components`
- [x] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.
When using either NVDA or VoiceOver in Firefox on Windows or Safari on macOS the below the fold content cannot be read. :hurtrealbad: 

I think we had issues with the reading order in this component in the past right @abbeyhrt ? 

> Is this issue related to a specific component?
tile

> What did you expect to happen? What happened instead? What would you like to
> see changed?

> What browser are you working in?
Firefox latest on Windows with NVDA, Safari latest on macOS

> What version of the Carbon Design System are you using?
latest -- testing on react.carbondesignsystem.com

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

## Steps to reproduce the issue

1. go to the expandable tile
2. using voiceover navigate to the tile and hear the above the fold content read
3. navigate to the button and expand the tile
4. attempt to navigate backwards to read below the fold content, observe you cannot

cc @snidersd

**Comment by abbeyhrt** [comment]
@dakahn I checked with VO in Chrome, Safari, and Firefox and I can get to the below the fold content navigating forward. Is the expectation that a user would navigate backwards to get to the below the fold content?

**Comment by dakahn** [comment]
Yeah reading order should be the same in reverse. That seems to be the requirement right, @snidersd ?

**Comment by snidersd** [comment]
@dakahn Correct, the reading order should be the same in reverse and the user should be able to expand, read and collapse in a logical order.

**Comment by aledavila** [comment]
Hi 2 developers worked through this and is working as expected and had the expected behavior. Using arrow keys you are able to navigate the component and get to above and below content as expected after opening. Closing
